src/compiler/transformers/declarations/emitBinder.ts

```diff
@@ -369,171 +435,280 @@ export function bindSourceFileForDeclarationEmit(file: SourceFile) {
             else {
                 Debug.assertNever(d.name);
             }
+            function isExportedVariable(d: VariableDeclaration | ParameterDeclaration) {
+                if (!isVariableDeclaration(d)) return false;

```

Comment:
Rather do `if(isParameterDeclaration(d)) return false` to be more explicit?

Reply:
Yeah. that sounds better. Changed